src/Core/System/Window.cpp

```diff
@@ -193,6 +193,14 @@ namespace obe::system
         std::string title = "ObEngine";
         if (configuration.contains("title"))
             m_title = configuration.at("title");
+
+        event::EventNamespace event_namespace = event::EventNamespace("obe::system");
```

Comment:
Check out : https://github.com/ObEngine/ObEngine/blob/6ad69858c82733b798881cd9fe4b0a90e2af52f2/src/Core/Engine/Engine.cpp#L82

for how to create a `EventNamespace`, although that made me realize that the constructor should be private, I'll fix this :)

Reply:
Ok I see this, do let me know when you have your changes in place for the constructor.

Reply:
Should be good, rebase your branch on latest v0.5 commit :)